README.markdown

```diff
@@ -1574,6 +1576,43 @@ This hook no longer runs in the cache manager and cache loader processes since t
 
 [Back to TOC](#directives)
 
+exit_worker_by_lua_block
+------------------------
+
+**syntax:** *exit_worker_by_lua_block { lua-script }*
+
+**context:** *http*
+
+**phase:** *exiting-worker*
+
+Runs the specified Lua code upon every Nginx worker process's exit when the master process is enabled. When the master process is disabled, this hook will before run Nginx process exiting.
```

Comment:
typo: will run before the Nginx process exits.

Reply:
@thibaultcha fixed, and sorry for these low-level error.